conans/client/remover.py

```diff
@@ -75,9 +75,7 @@ def remove_packages(self, package_layout, ids_filter=None):
                 pref = PackageReference(package_layout.ref, package_id)
                 if not package_layout.package_exists(pref):
                     raise PackageNotFoundException(pref)
-                pkg_folder = package_layout.package(pref)
                 package_layout.package_remove(pref)
-                self._remove_file(pkg_folder + ".dirty", package_layout.ref, "dirty flag")
                 self._remove_file(package_layout.system_reqs_package(pref), package_layout.ref,
                                   "%s/%s" % (package_id, SYSTEM_REQS))
```

Comment:
Does it make sense to remove this file too when the package_id is removed?

Reply:
I mean, remove this file inside the `layout.package_remove` method

Reply:
Yeah, I didn't want to touch system-requirements in this PR, I prefer doing 1 thing at a time, that will follow in another PR